include/circt/Dialect/FIRRTL/FIRRTLCanonicalization.td

```diff
@@ -70,22 +70,29 @@ def MuxSameCondHigh : Pat<
     (EqualTypes $x, $y), (EqualTypes $x, $z), (KnownWidth $x)
   ]>;
 
-// node(x, name, {}) -> x
+// node(x) -> x
 def EmptyNode : Pat<
   (NodeOp $x, $name, $annotations),
   (replaceWithValue $x),
   [(EmptyAttrDict $annotations)]>;
 
-// regreset(clock, invalidvalue, resetValue, name, {}) -> reg(clock, name, {})
+// regreset(clock, invalidvalue, resetValue) -> reg(clock)
 def RegresetWithInvalidReset : Pat<
   (RegResetOp $clock, (InvalidValueOp), $resetValue, $name, $annotations),
   (RegOp $clock, $name, $annotations),
   []>;
 
+// regreset(clock, reset, invalidvalue) -> reg(clock)
```

Comment:
Can you split out canonicalizations?  This pr is touching a lot of files.

Reply:
Done!